tor_ocr/main.py

```diff
@@ -36,6 +38,17 @@
 
     u_tor_post_id.reply(ocr_magic)
 """
+dotenv.load_dotenv()
+# TODO: set the client credentials to environment variables
+b_api = BlossomAPI(
+    email=os.environ.get('TOR_OCR_EMAIL'), 
+    password=os.environ.get('TOR_OCR_PASSWORD'), 
+    api_key='TOR_OCR_BLOSSOM_API_KEY', 
+    api_base_url=os.environ.get('TOR_OCR_BLOSSOM_API_BASE_URL')
+)
+
+api_base_url: str=os.environ.get('TOR_OCR_BLOSSOM_API_BASE_URL'),
```

Comment:
Do these lines need to be here or were they references?

Reply:
reference, should be removed now.